qa/tasks/cephfs/caps_helper.py

```diff
@@ -12,6 +12,106 @@
 log = getLogger(__name__)
 
 
+def gen_mon_cap_str(caps):
```

Comment:
IMO, if the intent is to add helpers, then this can be implemented as an abstract class with subclasses providing the appropriate implementations.